installing/installing_azure/manually-creating-iam-azure.adoc

```diff
@@ -18,8 +18,8 @@ include::modules/manually-create-identity-access-management.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../updating/updating-cluster-within-minor.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Updating a cluster using the web console]
-* xref:../../updating/updating-cluster-cli.adoc#manually-maintained-credentials-upgrade_updating-cluster-cli[Updating a cluster using the CLI]
+* xref:../../updating/updating_a_cluster/updating-cluster-web-console.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Updating a cluster using the web console]
```

Comment:
I was planning to save more extensive refactoring for another PR, and probably will nonetheless, but I went ahead and changed this single assembly's ID and context to be `updating-cluster-web-console` instead of `updating-cluster-within-minor`, and updated all links accordingly.

I'll go ahead and do the same thing with the EUS-to-EUS page's IDs as well.